src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobWriter.cs

```diff
@@ -431,7 +431,7 @@ public void WriteSerializedString(string str)
         /// The string is UTF16 encoded and prefixed by the its size in bytes.
         ///
         /// This final byte holds the value 1 if and only if any UTF16 character within the string has any bit set in its top byte,
-        /// or its low byte is any of the following: 0x01–0x08, 0x0E–0x1F, 0x27, 0x2D, 0x7F. Otherwise, it holds 0.
+        /// or its low byte is any of the following: 0x01-x08, 0x0E-x1F, 0x27, 0x2D, 0x7F. Otherwise, it holds 0.
```

Comment:
was the leading zero intentionally deleted?  (I'd expect it to say "0x01-0x08" still, not "0x01-x08"

Reply:
Nope.  That was a typo.  Will fix.